### PR TITLE
Add documentation for the pastetext toolbar button for the Paste plugin

### PR DESCRIPTION
**Pull request by albinsunnanbo**
Related Ticket: 

Description of Changes:
Add documentation for the pastetext toolbar button for the Paste plugin

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: plugins/paste.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ tinymce.init({
   selector: "textarea",  // change this value according to your HTML
   plugins: "paste",
   menubar: "edit",
-  toolbar: "paste"
+  toolbar: "paste pastetext" // paste corresponds to the `Paste` menu button, pastetext corresponds to the `Paste as text` menu button
```
Comment (lnewson) [inline plugins/paste.md:27]: The term `menu button` here is incorrect and actually means something else entirely. This should probably be `toolbar button` instead.

**Comment by lnewson** [comment]
LGTM as well, with the exception of the incorrect term in the comment. Thanks for the contribution @albinsunnanbo!

File: plugins/paste.md (hunk 1 above)
Comment (albinsunnanbo) [reply plugins/paste.md:27]: I was referring to that the toolbar button is the same as the command in the menu, but the comment apparently could be interpreted in several ways.
Maybe I should just remove the comment altogether, the toolbar button names are pretty self explainable.
The main reason for the documentation update was to make it easy do discover the pastetext toolbar button as I had to dig a bit to find that there was such a button.

Comment (tylerkelly13) [reply plugins/paste.md:27]: > Maybe I should just remove the comment altogether, the toolbar button names are pretty self explainable.

Might be the easiest option :)

Comment (albinsunnanbo) [reply plugins/paste.md]: Done, 
It's could be one my smallest PR ever :-)

**Comment by tylerkelly13** [comment]
Thanks again @albinsunnanbo. Update is now live on the documentation site: https://www.tiny.cloud/docs/plugins/paste/#example

**Pull request merged**